### DISCUSSION TITLE
Is there any possibility of Anime4k working on tvs or Nvdia Shield?

### DISCUSSION THREAD

**Discussion by AntoLupo** [bloc97/Anime4K#183] [General]
I am planning to setup a NAS server for playback with my TV or Nvidia Shield as a client. I am flexible with the app utilised for playback. I just want Anime4k or an equivalent solution to work for my anime. Is there something like this even if Anime4k isn't it? 

BTW great job, I can't even watch anime anymore without Anime4k. It just makes everything so much better. I use the A+A preset. CTRL+4.

**Comment by yooooooLeon** [comment]
您的邮件已收到！谢谢！

**Comment by bloc97** [comment]
I'm not sure if any other media player supports loading arbitrary GLSL shaders as well as mpv does. I don't know if nvidia shield supports external shaders as I'm not too familiar with it.

However I do know that mpv runs on android, though you will need a pretty powerful device to run the medium-sized shaders (i've succesfully used the S versions on a Snapdragon 855 SoC) Maybe if you can install mpv on the shield?

**Comment by Tama47** [comment]
I believe it would be possible to run Anime4K on an Nvidia Shield, but the experience might not be great. Since you're already planning to set up a NAS server, I recommend installing a Plex Media Server and using the Plex client app that supports mpv configuration. They offer a Plex HTPC client app for use with TV as well. I have added the guide in the [latest pull](https://github.com/Tama47/Anime4K/blob/patch-2/md/GLSL_Instructions_Plex.md).
